keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -1213,6 +1283,8 @@ Are there any tests that were run/should be run to understand performance charac
 and validate the declared limits?
 -->
 
+Yes, in the same way a large image used to run a container could use these resources (so no additional risk)
```

Comment:
Given that a huge interest in this is to help pull AI models, I do think that this would cause increased inodes/disk.

The additional risk (IMO) would be that in addition to images, we could potentially have really large images pulled into a volume.

Reply:
yeah my point is that this actually reduces that risk. the alternative (distribute a model through oci) would be to pack it into the container image, and that's prone to not be shared on the node. There's no limit to the size of an image, so this risk is present already and is only improved with this feature

Reply:
I see.